deployment/provider.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+  }
+  backend "s3" {
+    bucket = "trento-telemetry-terraform-backend"
+    key = "trento-telemetry"
```

Comment:
As we don't have the format checker here, this doesn't complain, but we should use the same indentation in the entries